.eslintrc.js

```diff
@@ -62,6 +77,8 @@ module.exports = {
 		COMMIT_SHA: true,
 		// this is when Webpack last built the bundle
 		BUILD_TIMESTAMP: true,
+		// this is for e2e tests
+		step: false,
```

Comment:
Can we set `globals` in the `overrides` for e2e above? If not, let's add an eslint config at the e2e root which contains all of the config.

I don't like the idea of adding a new global everywhere that isn't actually available and has a pretty generic name 🙂

Reply:
Yeah, good suggestion. Moved it to overrides and it works wonders. Thanks!